Palette self-hosted airgap refactor (#1230)
spectrocloud/librarium: content/docs/19-kubernetes-knowlege-hub.md

---
title: "Kubernetes Knowledge Hub"
metaTitle: "Kubernetes Knowledge Hub"
metaDescription: "Kubernetes Knowledge Hub Respository"
icon: "bookmark"
hideToCSidebar: true
hideToC: true
fullWidth: true
---

import Tabs from 'shared/components/ui/Tabs';
import WarningBox from 'shared/components/WarningBox';
import InfoBox from 'shared/components/InfoBox';
import PointsOfInterest from 'shared/components/common/PointOfInterest';
import Tooltip from "shared/components/ui/Tooltip";

# Welcome to the Spectro Cloud Knowledge Portal

> Welcome to the Spectro Cloud Kubernetes Knowledge Hub. You will find core Kubernetes tutorials, how-tos, frequently asked questions, and community curated resources.

Reviewer: ```suggestion
Welcome to the Spectro Cloud Kubernetes Knowledge Hub. You will find core Kubernetes tutorials, how-tos, frequently asked questions, and community-curated resources.
```

> If you have a topic in mind you would like to see, use the Feedback app on the lower-right-hand corner.

Reviewer: ```suggestion
If you have a topic in mind you would like to see, use the Feedback app at lower right.
```
